src/strings/eng.json

```diff
@@ -48,6 +48,7 @@
     "Help_Improve": "Help us improve Apertium!",
     "Contact_Us": "Feel free to contact us if you find a mistake, there's a project you would like to see us work on, or you would like to help out.",
     "Maintainer": "This website is maintained by {{maintainer}}.",
+    "More_Languages": "Looking for more languages? Try {{more_languages}}",
```

Comment:
Should this include a `.` at the end? It'll be easier to tell if you include a screenshot in the PR description.